frontend/pages/hosts/details/cards/About/_styles.scss

```diff
@@ -1,4 +1,38 @@
 .about-card {
+
+  .tooltip-truncated-text {
+    display: inline-block;
+
+  }
+
+  .truncated-tooltip {
+    .about-card__device-mapping__source {
+        color: inherit;
+      }
```

Comment:
so the source (e.g. `(google chrome)`) doesn't show up as grey on grey in the tooltip.